src/libecalc/dto/components.py

```diff
@@ -51,7 +51,7 @@
 def check_model_energy_usage_type(model_data: Dict[datetime, ConsumerFunction], energy_type: EnergyUsageType):
     for model in model_data.values():
         if model.energy_usage_type != energy_type:
-            raise ValueError(f"Model does not consume {energy_type}")
+            raise ValueError(f"Model does not consume {energy_type.value}")
```

Comment:
Do you know when this change happened? It's `str(someEnum)` that becomes `EnumName.name` instead of the string value right? Does that mean a string enum no longer behaves as a string?

Reply:
sorry, didnt see comment before I merged. It was introduced when we were supposed to make ecalc compatible with 3.8 -> 3.11. good point. not sure when this occurs ...